gpkit/tests/t_constraints.py

```diff
@@ -30,8 +30,8 @@ def test_additive_scalar(self):
         c2 = 1 >= 5*x + 0.5
         self.assertEqual(type(c1), PosynomialInequality)
         self.assertEqual(type(c2), PosynomialInequality)
-        self.assertEqual(c1.posylt1_rep.cs, c2.posylt1_rep.cs)
-        self.assertEqual(c1.posylt1_rep.exps, c2.posylt1_rep.exps)
+        self.assertEqual(c1.as_posyslt1()[0].cs, c2.as_posyslt1()[0].cs)
+        self.assertEqual(c1.as_posyslt1()[0].exps, c2.as_posyslt1()[0].exps)
```

Comment:
shall we use your 
`var, = thing_to_unpack`
trick here to also assert length 1?